runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -4410,6 +4410,11 @@ TR::CompilationInfoPerThreadRemote::getCachedResolvedMethod(TR_ResolvedMethodKey
       auto methodInfo = methodCacheEntry.methodInfo;
       TR_OpaqueMethodBlock *method = methodCacheEntry.method;
       uint32_t vTableSlot = methodCacheEntry.vTableSlot;
+      if (!method)
+         {
+         *resolvedMethod = NULL;
+         return true;
+         }
```

Comment:
I'd suggest re-ordering this so that it's a little easier to read. `methodInfo` and `vTableSlot` are only used below the code you're adding, so it's better to define them after the `if (!method) { ... }` block.